PopPUNK/network.py

```diff
@@ -1462,7 +1462,7 @@ def printClusters(G, rlist, outPrefix=None, oldClusterFile=None,
         for isolate_index, isolate_name in enumerate(rlist): # assume sorted at the moment
             component = component_assignments['labels'].iloc[isolate_index].item()
             component_rank_bool = component_frequencies.index == component
-            component_rank = np.argmax(component_rank_bool.to_array())
+            component_rank = int(cp.argmax(component_rank_bool))
```

Comment:
always correct to use `cp` rather than `np` here?

Reply:
Yep - what we had before crashed out